health-checker/src/main/java/io/stargate/health/CPUGaugeMetricSet.java

```diff
@@ -0,0 +1,97 @@
+package io.stargate.health;
+
+import com.codahale.metrics.Gauge;
+import com.codahale.metrics.Metric;
+import com.codahale.metrics.MetricSet;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+import oshi.SystemInfo;
+import oshi.hardware.CentralProcessor;
+import oshi.hardware.CentralProcessor.TickType;
+
+public class CPUGaugeMetricSet implements MetricSet {
```

Comment:
Tests unit OR/AND IT are missing. The logic is not trivial, please add tests. We can mock the CPU system provider, and test our logic.

Reply:
I'm just grabbing an array and adding some numbers together. Seems pretty straightforward to me.

Reply:
FWTW, we could use a unit test to make sure input tick numbers are wired to the right metric name.

Reply:
added